aptos-move/framework/aptos-framework/sources/configs/execution_config.spec.move

```diff
@@ -4,16 +4,7 @@ spec aptos_framework::execution_config {
         pragma aborts_if_is_strict;
     }
 
-    /// Ensure caller is admin.
-    spec initialize(aptos_framework: &signer, config: vector<u8>) {
```

Comment:
This only removes the spec. Can you remove that initialize function in execution.move as well?

Reply:
Ah, my bad. I had this change locally but didn't include that in the PR.

Reply:
Done!